Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -864,6 +863,56 @@
 }
 #endif
 
+std::optional<SimpleRange> WebPage::autocorrectionContextRange()
+{
+    // This uses the same algorithm as `WebPage::autocorrectionContext`.
```

Comment:
Can any code be shared?

Reply:
Looking at the code, it does look different enough, to the point where it can't be shared?

If that's the case, I would remove this comment and briefly describe the algorithm instead.